core/src/test/java/tech/tablesaw/joining/DataFrameJoinerTest.java

```diff
@@ -1438,6 +1439,46 @@ public void leftOuterJoinOnAgeMoveInDate() {
     assertEquals(9, joined.rowCount());
   }
 
+  @Test
+  public void leftOuterJoinOnAgeMoveInDateKeepAllJoinKeyColumns() {
```

Comment:
these method names are pretty long. `leftOuterJoin_keepAllJoinKeyColumns` might be more readable?

Reply:
OK, I will modify it later

Reply:
@benmccann I have already renamed the test name.